Core/MIPS/JitCommon/JitCommon.cpp

```diff
@@ -157,10 +159,12 @@ const char *ppsspp_resolver(struct ud*,
 		*offset = addr - (uint64_t)MIPSComp::jit->GetBasePtr();
 		return "jitcode";
 	}
+#if defined(_M_IX86) || defined(_M_X64) || defined(MIPS)
```

Comment:
It was a synthetic test to catch issues that may affect [Tier2](https://www.freebsd.org/doc/en/articles/committers-guide/archs.html#idp59530064) architectures like powerpc, sparc64. FreeBSD package builds are irregular for  them, so such (trivial) issues may not be noticed until months or years later.
```
CMAKE_ARGS+=-DX86=off
CFLAGS+=-U__i386__ -U__x86_64__
```